vpdq/python/vpdq.pyx

```diff
@@ -98,7 +99,7 @@ def hamming_distance(hash1: "Hash256", hash2: "Hash256") -> int:
 
 
 def computeHash(
-    input_video_filename: str,
+    input_video_filename: t.Union[str, Path],
```

Comment:
Why not just accept one?

Reply:
I think accepting two will make the function easier to use.